### DISCUSSION TITLE
TEST RESULTS  in hammerdbcli are missing

### DISCUSSION THREAD

**Discussion by tingjiec** [TPC-Council/HammerDB#129] [Q&A] [answered]
I setup hammerdb with v3.3 and mysql, in hammerdbcli as following:
dbset db pg
dbset bm tpc-c
vuset logtotemp 1
vuset showoutput 1
buildschema
vurun

but the output shows on "TEST RESULT", does the feature on cli have removed?
---------------------------------------------
Vuser 1:order status
Vuser 2:new order
Vuser 1:2169 OUGHTOUGHTCALLY nxqaQ0vmVzu1Nmq OE 118796.00 54737 2020-06-11
Vuser 2:0.0000 OUGHTPRESANTI BC 0.1100 0.1600 54642
Vuser 1:stock level
Vuser 2:new order
Vuser 1:1 2 17
Vuser 2:0.2600 PRIBARABLE BC 0.1100 0.1600 54553
Vuser 1:new order
Vuser 2:payment
Vuser 2:2681 OUGHTABLEEING ijVuxqTS9uSDXnpS mCiZziOTHdHnrp c2CJt3Pg66Vr QP 240211111 ghgULkLpzH5IXi YROzsKekLbd0Mr0x vsZ89113xC Wk 159311111 m6etB1BH OE TUE1T4Jd42jtOwvnW 0qmwp4vieal20T8Bj 6ePKck2RhW6HLfNBRc7Y Ud 785211111 6379285266465399 2020-06-11 GC 50000.00 0.1600 235060.00
Vuser 1:0.1300 BAROUGHTBAR BC 0.1100 0.1600 54554
Vuser 1:new order
Vuser 2:new order
Vuser 2:0.3000 BARPRESANTI BC 0.1000 0.1600 54007
Vuser 1:0.1400 OUGHTCALLYCALLY GC 0.1000 0.1600 54513
Vuser 2:payment
Vuser 1:stock level
Vuser 1:1 2 18
Vuser 2:2862 ANTIPRIPRES ijVuxqTS9uSDXnpS mCiZziOTHdHnrp c2CJt3Pg66Vr QP 240211111 Sj1Oa7uNQrx 8N5rs3h9zSveMJqFw 7XcQRuuVMO0 eS 605111111 WOb50eT9IpO OE cOMn6DyDUTH5xy8VG jqj3ebaaZKdcHVf5 xs0mvQ4FT3zcg QA 406511111 3272283672710964 2020-06-11 GC 50000.00 0.1700 2121292.00
Vuser 1:new order
Vuser 2:new order
          Timer: 2 minutes elapsed
Vuser 1:0.1700 ABLEABLEPRI GC 0.1800 0.1600 54428
Vuser 2:0.0200 PRICALLYCALLY BC 0.1100 0.1600 54643
          Destroying Virtual Users
Vuser 2:new order
Vuser 1:FINISHED SUCCESS
          Virtual Users Destroyed
Vuser 2:FINISHED SUCCESS
ALL VIRTUAL USERS COMPLETE
TPC-C Driver Script
vudestroy success
TEST SEQUENCE COMPLETE

**Comment by sm-shaw** [comment, marked as answer]
You are running the test driver script, in both the GUI and CLI the test driver script does not report a result. The test driver script is a simpler script that can be used to create a workload when you are not aiming to measure performance and instead want to look at functionality. To switch to the timed script in the CLI for PostgreSQL you need to do run the command:
`hammerdb>diset tpcc pg_driver timed`
Note that all of the options you see by running:
`print dict`
correspond to an option in the GUI interface, the GUI will guide you towards the right options to select whereas the CLI won't and therefore the CLI is intended for more advanced users who have some experience of using the GUI.  Even if you are testing a server without a GUI environment running the HammerDB GUI locally on a PC can help guide you to choosing the right options.

**Comment by tingjiec** [comment]
> You are running the test driver script, in both the GUI and CLI the test driver script does not report a result. The test driver script is a simpler script that can be used to create a workload when you are not aiming to measure performance and instead want to look at functionality. To switch to the timed script in the CLI for PostgreSQL you need to do run the command:
> `hammerdb>diset tpcc pg_driver timed`
> Note that all of the options you see by running:
> `print dict`
> correspond to an option in the GUI interface, the GUI will guide you towards the right options to select whereas the CLI won't and therefore the CLI is intended for more advanced users who have some experience of using the GUI. Even if you are testing a server without a GUI environment running the HammerDB GUI locally on a PC can help guide you to choosing the right options.

Thanks very much, I used it for the first time, and read more document for it, close the issue ...
